apps/admin-x-activitypub/src/views/Profile/Profile.tsx

```
@@ -4,24 +4,23 @@ import Posts from './components/Posts';
-import {useAccountFollowsForUser, useAccountForUser, usePostsByAccount, usePostsLikedByAccount, useProfileFollowersForUser, useProfileFollowingForUser, useProfilePostsForUser} from '@hooks/use-activity-pub-queries';
+import {useAccountFollowsForUser, useAccountForUser, usePostsByAccount, usePostsLikedByAccount, useProfileFollowersForUser, useProfileFollowingForUser} from '@hooks/use-activity-pub-queries';
 import {useParams} from '@tryghost/admin-x-framework';
 
 export type ProfileTab = 'posts' | 'likes' | 'following' | 'followers';
 
 interface ProfileProps {}
 
-const PostsTab:React.FC<{handle?: string, currentUserHandle: string}> = ({handle, currentUserHandle}) => {
-    const postsByAccountQuery = usePostsByAccount({enabled: true}).postsByAccountQuery;
-    const profilePostsQuery = useProfilePostsForUser('index', handle || currentUserHandle);
+const PostsTab:React.FC<{}> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)